src/pnpq/devices/odl_thorlabs_kbd101.py

```diff
@@ -4,10 +4,32 @@
 #       Stage:                  DDS100/M
 #
 import serial
-import time
+import time, logging
 from serial import Serial
 from pnpq.devices.optical_delay_line import OpticalDelayLine
 
+from pnpq.errors import (
+    DevicePortNotFoundError,
+    DeviceDisconnectedError,
+    OdlMoveNotCompleted,
+    OdlHomeNotCompleted,
+    OdlGetPosNotCompleted,
+    OdlMoveOutofRangeError,
+)
+
+ODL_HOME_COMMAND = b"\x43\x04\x01\x00\x50\x01"
+ODL_MOVE_COMMAND = b"\x53\x04\x06\x00\xd0\x01\x00\x00"
+STOP_UPDATE_COMMAND = b"\x12\x00\x00\x00\x50\x01"
+START_UPDATE_COMMAND = b"\x11\x00\x00\x00\x50\x01"
+ODL_IDENTIFY_COMMAND = b"\x23\x02\x00\x00\x50\x01"
+ENABLE_CHANNEL_SET_COMMAND = b"\x10\x02\x01\x01\x50\x01"
+ENABLE_CHANNEL_GET_COMMAND = b"\x11\x02\x01\x00\x50\x01"
+ODL_RELATIVE_MOVE_COMMAND = b"\x48\x04\x06\x00\xd0\x01\x00\x00"
+MGMSG_MOT_REQ_USTATUSUPDATE = b"\x90\x04\x01\x00\x50\x01"
+MGMSG_MOT_GET_USTATUSUPDATE = b"\x91\x04"
```

Comment:
what do "MGMSG" and "MOT" mean? if it comes from some reference, we should put the link for it.

Reply:
yes. that comes from the reference. It's a special command message for a subset of motorized devices to update their status.